src/backend/python_backend/training_routes.py

```
@@ -63,7 +63,9 @@ async def start_training(
 
 @router.get("/api/training/status/{job_id}")
 @log_performance(operation="get_training_status")
-async def get_training_status(job_id: str, current_user: str = Depends(get_current_active_user)):
+async def get_training_status(
+    job_id: str, current_user: str = Depends(get_current_active_user)
```

Flake8 (7.3.0)

[error] undefined name 'Depends'

(F821)